qiskit/backends/ibmq/api/ibmqconnector.py

```diff
@@ -590,7 +590,12 @@ def get_job(self, id_job, hub=None, group=None, project=None,
 
         job = self.req.get(url)
 
-        if 'qasms' in job:
+        if 'qObjectResult' in job:
```

Comment:
Has something changed in the backend? Why is this change necessary?

Reply:
This is a cherrypick of #1263.